kettle-integration/core/pom.xml

```diff
@@ -22,61 +22,44 @@
     <dependency>
       <groupId>pentaho</groupId>
       <artifactId>pentaho-data-profiling-model-core</artifactId>
-      <version>${project.version}</version>
     </dependency>
     <dependency>
       <groupId>pentaho</groupId>
       <artifactId>pentaho-data-profiling-api-core</artifactId>
-      <version>${project.version}</version>
     </dependency>
     <dependency>
       <groupId>pentaho-kettle</groupId>
       <artifactId>kettle-ui-swt</artifactId>
-      <version>${kettle.version}</version>
-      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>pentaho-kettle</groupId>
       <artifactId>kettle-engine</artifactId>
-      <version>${kettle.version}</version>
-      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>pentaho-kettle</groupId>
       <artifactId>kettle-core</artifactId>
-      <version>${kettle.version}</version>
-      <scope>provided</scope>
```

Comment:
@JoaoLMPereira this needs to be provided for the project to build in intellij (it has classes in the default package)